src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CSharpMetadataContext.cs

```diff
@@ -1,26 +1,35 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System.Collections.Immutable;
+using System.Linq;
 using Microsoft.CodeAnalysis.ExpressionEvaluator;
 
 namespace Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator
 {
-    internal sealed class CSharpMetadataContext : MetadataContext
```

Comment:
If it's just refactoring to make struct, consider whether the additional indirection is worth the anticipated cost of the allocations (I assume we don't usually allocate more than a few of these per debug session).  At least for me, the extra indirection makes the code dealing with this type harder to read.